packages/eds-core-react/src/components/Icon/Icon.docs.mdx

```diff
@@ -59,6 +67,12 @@ If you are using the icons alone to convey meaning you need to provide the `titl
 
 
 ## Sizes
+Simple and clear icons require a strict underlying grid.
+<ul>
+    <li>The EDS icons are built on a 24x24px grid.</li>
+    <li>System icons can be used in 24px and 16px sizes.</li>
```

Comment:
Suggest removing these two paragraphs as the sizes in props table speaks for them selves. Good to mention when its recommended to use 18 px as its the odd one out